.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 language: node_js
 node_js:
   - 4
-  - 5
+  - stable
```

Comment:
why? it is just additional time, and I have not seen a single thing working in 4 & 6 that could break in 5